src/cf.data.pre

```diff
@@ -181,6 +181,13 @@ DOC_START
 	This option is not yet supported by Squid-3.
 DOC_END
 
+# Options removed in 5.x
+NAME: dns_v4_first
+TYPE: obsolete
+DOC_START
+	Remove this line. Squid performs a 'Happy Eyeballs' algorithm, doing v4 first is no longer relevant.
```

Comment:
Preferring IPv4 over IPv6 addresses is still relevant (to Squid admins -- the intended audience of this documentation). We just do not have the code to enable some form of such preferential treatment. I suggest the following adjustment to avoid misleading or upsetting admins, but you may want to get Amos consent before applying this suggestion.

```suggestion
	Remove this line. Squid no longer supports preferential treatment of DNS A records.
```

@yadij, any objections to this change?

Reply:
fine by me